erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -795,3 +796,36 @@ def update_item_quantity(source, target, source_parent):
 	doc.set_item_locations()
 
 	return doc
+
+@frappe.whitelist()
+def start_job_cards(name):
+	job_cards = []
+	open_job_cards = frappe.db.get_all('Job Card', {
+		'work_order':name, 'status':('in', ['Open', 'Work In Progress', 'Material Transferred'])
+		}, ['*'])
+	for job_card in open_job_cards:
+		job = frappe.get_doc('Job Card', job_card.name)
+		job.job_started = 1
+		row = job.append('time_logs', {})
+		row.from_time = now_datetime()
+		row.completed_qty = 0
+		job.save()
+		job_cards.append(job.name)
+	return job_cards
+
+@frappe.whitelist()
+def stop_job_cards(name):
+	job_cards = []
+	open_job_cards = frappe.db.get_all('Job Card', {
+		'work_order':name, 'status':('in', ['Open', 'Work In Progress', 'Material Transferred'])
+		}, ['*'])
+	for job_card in open_job_cards:
+		job = frappe.get_doc('Job Card', job_card.name)
+		job.job_started = 0
+		for row in job.time_logs:
+			if not row.to_time:
+				row.to_time = now_datetime()
+				row.completed_qty = 0
+		job.save()
+		job_cards.append(job.name)
```

Comment:
Append a linkable string so that it can be displayed on the frontend, use `frappe.utils.get_link_to_form` for that

Reply:
@vishdha, you missed out on this one